tracer/build/_build/Build.Profiler.Steps.cs

```diff
@@ -61,4 +64,26 @@ partial class Build
                 .CombineWith(platforms, (m, platform) => m
                     .SetTargetPlatform(platform)));
         });
+    Target CompileProfilerNativeSrcLinux => _ => _
+        .Unlisted()
+        .After(CompileProfilerManagedSrc)
+        .OnlyWhenStatic(() => IsLinux)
+        .Executes(() =>
+        {
+            var buildDirectory = RootDirectory / ".." / "_build" / "cmake";
```

Comment:
Is this really the right directory 🤔 This starts from the top level root directory, then goes _up_? _Everything_ should be inside `RootDirectory` right?

EDIT: Apparently not, because the profiler is separate. In any case, we should probably be inside `ProfilerDirectory` instead?
```suggestion
            var buildDirectory = ProfilerDirectory / "_build" / "cmake";
```

Reply:
I build at the profiler in directory at the same level as the profiler source tree because that's what they do in their CI: https://github.com/DataDog/dd-continuous-profiler-dotnet/blob/master/.github/workflows/buid_and_test.yml#L48

It shouldn't be a problem to move it, but wanted to keep it consistent with what they do, but I don't know how important that is 🤷

Reply:
I think they build outside the tree because cmake can produce lots of different files that are not always .gitignored correctly.

Reply:
> I think they build outside the tree because cmake can produce lots of different files that are not always .gitignored correctly.

Fair enough, feels like the correct solution is to be gitignoring the build directory, but that's a profiler issue, so may as well leave it as-is for now